openassessment/assessment/api/peer.py

```diff
@@ -565,7 +565,7 @@ def get_submitted_assessments(submission_uuid, scored_only=True, limit=None):
 
     Keyword Arguments:
         scored (boolean): Only retrieve the assessments used to generate a score
-            for this submission.
+            for their respective submission.
```

Comment:
Argument name is "scored_only". I'm not sure about the doc string-- is this method used? Is "scored_only" used? The whole thing is confusing since the doc says "this API returns related feedback without making any assumption about grading". 

I feel like there is much sketchiness with this method, and I'm not sure it is worth trying to fix the doc without understanding the method.